pages/faq/troubleshooting/troubleshooting.md

```diff
@@ -72,7 +72,7 @@ If the Raspberry pi is unable to connect to the {{ $names.company.lower }} serve
 This is either because it is not connected to the network or because the network ports which {{ $names.company.lower }} relies on are blocked in some way.
 
 * The first things to check in this case is that your device is correctly connected to ethernet or that you correctly entered the wifi credentials. To check wifi credentials, power your device down, remove the SD card, and mount the SD card on your personal computer. If your device is running {{ $names.os.lower }} version 2.0 or greater, wifi credentials are listed in `system-connections/resin-wifi`, found in the `resin-boot` partition of the SD card. Otherwise, check the `config.json` file (in the `resin-boot` partition for versions 1.2 and greater, or `resin-conf` for earlier versions).
-* Secondly check that your network is not restricting or blocking the ports specified in the [{{ $names.company.lower }} network requirements](/deployment/network/#network-requirements).
+* Secondly check that your network is not restricting or blocking the ports specified in the [{{ $names.company.lower }} network requirements](/reference/OS/network/2.x/#network-requirements).
```

Comment:
It would be nice to be able to avoid a reference to a specific version number (2.x). Perhaps a way of defaulting to the latest version.